### ISSUE TITLE
Is this project still being updated?

### ISSUE THREAD

**Issue by mikewojtkiewicz** [Kirollos/Oxide.Ext.Discord#53]
The Discord API has changed a little bit since this has a check in last. I was curious if this was still being updated.

**Comment by dassjosh** [comment]
Yes this is. Checkout the develop branch for Discord Extension 3.0's ongoing development.

**Comment by mikewojtkiewicz** [comment]
I'm looking for the GlobalName property on the DiscordUser object, and it doesn't appear to be there. I created a PR for it on your fork
